generators/generator-multibot/README.md

```diff
@@ -0,0 +1,32 @@
+# generator-conversational-core [![NPM version][npm-image]][npm-url]
```

Comment:
Should replace contents with the multibot instructions

Reply:
Yeah and get rid of this string '[![NPM version][npm-image]][npm-url]' It will throw errors when compser pulls down the markdown for the readme

Reply:
good idea, i included a link

Reply:
I have the other READMEs using this format, I'll fix them

Reply:
Looks like it's already done, thanks Pat :)